ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/mapper/ClaimantMapperTest.java

```diff
@@ -54,7 +58,7 @@ public void shouldMapOrganisationToCCD() {
         Party party = SampleParty.builder().organisation();
 
         //when
-        CCDClaimant ccdParty = claimantMapper.to(party);
+        CCDClaimant ccdParty = claimantMapper.to(party, claim);
```

Comment:
ClaimantMapper has changes 

builder.partyEmail(claim.getSubmitterEmail());

But its not tested. Please add this assertion

Reply:
there is no object call ed ccdClaimData, but a ccdClaim, so assertion is covered in CaseMapperTest